app/controllers/concerns/part_of_claim_journey.rb

```diff
@@ -12,6 +13,16 @@ def current_policy_routing_name
     current_claim.policy&.routing_name
   end
 
+  def check_for_maintenance_mode
+    policy = Policies[current_policy_routing_name]
+    policy_configuration = PolicyConfiguration.find_by(policy_type: policy.name)
+
+    if policy_configuration.maintenance_mode?
+      @availability_message = policy_configuration.maintenance_mode_availability_message
+      render "static_pages/maintenance", status: :service_unavailable
+    end
+  end
```

Comment:
Now that we're no longer handling maintenance mode in the router, this action method is redundant as we only ever render maintenance mode from the `check_for_maintenance_mode` before action.